src/app-bridge/fetch.ts

```diff
@@ -0,0 +1,31 @@
+import { SALEOR_AUTHORIZATION_BEARER_HEADER, SALEOR_DOMAIN_HEADER } from "../const";
+import { AppBridge } from "./app-bridge";
+import { useAppBridge } from "./app-bridge-provider";
+
+type HasAppBridgeState = Pick<AppBridge, "getState">;
+export const createAuthenticatedFetch =
```

Comment:
How next ssr would handle the lack of window? 

Would be a good idea to add docstring explaining that is client-side only

Reply:
Good catch. You can't use it because there is no AppBridge. I replaced window with global.

We probably should assume in `_app.tsx` that apps are client-side only, server context doesn't make sense. I can document this